CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 - Add imperial support for UDCF import
 - Desktop: combo box with fresh, sea water and EN 13319 in dive planner
 - Desktop: Changed "salinity" to "water type" at dive planner and dive info
+- Desktop: Small UI fixes
```

Comment:
perhaps we should be more specific in place of "Small UI fixes"?

Reply:
I thought we wrote exactly the same also in the past?
Ok, if we want to have it more detailed I may need help. What exactly did we do?
- Sort the dive computer models in the configure DC dialog
- S.th. else?

Hmmm,... after writing this I still believe we better stay with "Small UI fixes" ;-)

Reply:
i don't like "small UI fixes". it covers a lot and tells nothing to the end user as we do "small UI fixes" every version bump. just had a look at the Git log - better remove this.

also, i've tried proposing a formatting standard, but it seems that's a lost cause.